### PR TITLE
[2293] Add mimetype list to the firewall docs

### LINKED ISSUE #2293: add our mimetype mapping to the firewall document
https://github.com/owncloud/core/blob/master/resources/config/mimetypemapping.dist.json

this file contains helpful mimetypes which are needed if you want to block upload of a certain type via the file firewall app.

We need to add this to the firewall config document.

https://doc.owncloud.com/server/admin_manual/enterprise/firewall/file_firewall.html

### PR DESCRIPTION
**Pull request by settermjd**
This fixes #2293.

### CODE REVIEW

**Comment by settermjd** [comment]
@voroyam, is this what you were after?

**Comment by voroyam** [comment]
> @voroyam, is this what you were after?

not really. 

Simple minded as I am, I would just copy the most popular extensions ( top 5 of them) like pdf, exe, mp4, mp3, .iso and the mime type to them. So that the usual use case is covered, and if you are looking for something more exotic you can check out the full list.

But the way you have done it is okay, maybe I do my version of it in a new PR :)

File: modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc [hunk 1]
```diff
@@ -1,42 +1,36 @@
...
-The File Firewall GUI enables you to manage firewall rule sets. You can
-find it in your ownCloud admin page, under `Admin -> Security`. The File
-Firewall lets you control access and sharing in fine detail, by creating
-rules for allowing or denying access restrictions based on: _group_,
-_upload size_, _client devices_, _IP address_, _time of day_, as well as
-many more criteria. In addition to these restriction options, the File
-Firewall app also supports rules based on
-http://www.regular-expressions.info/[regular expressions].
+The File Firewall GUI enables you to manage firewall rule sets. 
+You can find it in your ownCloud admin page, under `Admin -> Security`.
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc]: `Admin -> Security` --> why not using the menu macro as we do in the other documents?

Comment (settermjd) [reply modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc]: Good point.

Comment (settermjd) [reply modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc]: Change made anyway.

**Pull request merged**